jobboard/models.py

```diff
@@ -51,7 +51,11 @@ def get_jobs_by_city_name(cls, city_name):
         """
         Allows students to search for a job based on the city it is located in
         """
-        return set(cls.objects.filter(city=City.objects.get(name=city_name)))
+        try:
```

Comment:
I think you can do all of this function in one line like so:
`return set(cls.objects.filter(city__name__iexact=city_name))`

Reply:
Done